### PR TITLE
{Doc} Add data collection statement

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**

Add the required [Telemetry Notice](https://docs.opensource.microsoft.com/content/releasing/index.html) for an open source project, including:
- instructions to turn off the telemetry;
- data collection notice in the README.md file.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@jiasli @zhoxing-ms for awareness

File: README.md [hunk 1]
```diff
@@ -105,6 +105,14 @@ Please check [Tips for using Azure CLI effectively](https://docs.microsoft.com/e
 
 For more usage examples, take a look at our [GitHub samples repo](http://github.com/Azure/azure-cli-samples) or [https://docs.microsoft.com/cli/azure/overview](https://docs.microsoft.com/cli/azure/overview).
 
+## Data Collection
```
Comment (yungezz) [inline README.md:108]:
```suggestion
## Telemetry
```

File: README.md [hunk 2]
```diff
@@ -105,6 +105,14 @@ Please check [Tips for using Azure CLI effectively](https://docs.microsoft.com/e
...
+
+The software may collect information about you and your use of the software and send it to Microsoft. Microsoft may use this information to provide services and improve our products and services. You may turn off the telemetry as described in the repository. There are also some features in the software that may enable you and Microsoft to collect data from users of your applications. If you use these features, you must comply with applicable law, including providing appropriate notices to users of your applications together with a copy of Microsoft's privacy statement. Our privacy statement is located at https://go.microsoft.com/fwlink/?LinkID=824704. You can learn more about data collection and use in the help documentation and our privacy statement. Your use of the software operates as your consent to these practices.
+
+### Telemetry Configuration
+
+The collection of telemetry is currently enabled by default. If you want to turn it off, you can run `az config set core.collect_telemetry=no`. You can enable it back to help us improve our product by running `az config set core.collect_telemetry=yes`.
```
Comment (yungezz) [inline README.md:114]:
```suggestion
Telemetry collection is on by default. To opt out, pls run `az config set core.collect_telemetry=no` to turn if off.
```

File: README.md (hunk 1 above)
Comment (fengzhou-msft) [reply README.md:108]: The `Data Collection` title follows the [instruction](https://docs.opensource.microsoft.com/content/releasing/telemetry.html)

Comment (yungezz) [reply README.md:108]: lesson learnt. Telemetry is used widely in vscode extensiosn repos,  learnt from there :)

**Review by dbradish-microsoft — APPROVED** [review]

**Review by yungezz — APPROVED** [review]

**Pull request merged**